### PR TITLE
Temp8 4257 refactoring of infinite editor layers

### PR DESCRIPTION
**Pull request by nielslyngsoe**
Regarding issue #4257 I have refactored the way infinity editing layers gets added and removed, without lookups of the DOM.

### CODE REVIEW

**Comment by warrenbuckley** [comment]
Kudos @nielslyngsoe for fixing this up & not relying on DOM elements anymore 🎉

File: src/Umbraco.Web.UI.Client/src/common/directives/components/editor/umbeditors.directive.js [hunk 1]
```diff
@@ -13,145 +13,145 @@
...
+                    _tx: [100, 0],
+                    //opacity: [0, 1],
+                    easing: 'easeOutExpo',
+                    duration: 480,
+                    update: () => {
+                        editor.style.transform = "translateX("+editor.style._tx+"%)";
+                        scope.$digest();
+                    },
+                    complete: function() {
+                        //$timeout(function(){
```
Comment (Shazwazza) [inline src/Umbraco.Web.UI.Client/src/common/directives/components/editor/umbeditors.directive.js]: Lets try not to keep commented out code in the core please

File: src/Umbraco.Web.UI.Client/src/less/components/editor/umb-editor.less [hunk 2]
```diff
@@ -17,13 +17,14 @@
 }
 
 .umb-editor--animating {
-    will-change: transform, opacity, width, left;
+    //will-change: transform, opacity, width, left;
```
Comment (Shazwazza) [inline src/Umbraco.Web.UI.Client/src/less/components/editor/umb-editor.less]: No need to keep commented out code like this, that's what source control is for :) There's a couple of these in this PR

Comment (nielslyngsoe) [reply src/Umbraco.Web.UI.Client/src/less/components/editor/umb-editor.less]: Sorry, my mistake. I where suppose to get them out of there.

**Pull request merged**